lib/rgeo/geos/capi_feature_classes.rb

```diff
@@ -9,8 +9,8 @@
 require_relative "../impl_helper/validity_check"
 
 module RGeo
-  module Geos
-    module CAPIGeometryMethods
+  module Geos # :nodoc:
+    module CAPIGeometryMethods # :nodoc:
```

Comment:
yeah I think we should get rid of the `nodoc`s as well. Maybe we can add some docs too, but at least for now this should generate the docs for it with methods.

Reply:
I have reverted these changes add added back the rubocop todos as this will probably need more discussion and someone with more knowledge than me to fill in those.